client/my-sites/upgrades/checkout/secure-payment-form.jsx

```diff
@@ -62,22 +66,35 @@ const SecurePaymentForm = React.createClass( {
 			return 'free-trial';
 		} else if ( this.state && this.state.userSelectedPaymentBox ) {
 			return this.state.userSelectedPaymentBox;
-		} else if ( cartValues.isCreditCardPaymentsEnabled( cart ) ) {
-			return 'credit-card';
-		} else if ( cartValues.isPayPalExpressEnabled( cart ) ) {
-			return 'paypal';
+		} else if ( cartValues.isPaymentMethodEnabled( cart, preferredPaymentMethods.primary ) ) {
+			return preferredPaymentMethods.primary;
+		} else if ( cartValues.isPaymentMethodEnabled( cart, preferredPaymentMethods.secondary ) ) {
+			return preferredPaymentMethods.secondary;
 		}
 
 		return null;
 	},
 
+	getLocalizedPaymentMethodDefaults( locale, countryCode ) {
+		const default_payment_methods = { primary: 'credit-card', secondary: 'paypal' };
+		const defaults = {
```

Comment:
I think, in the long run, yes. But it feels like overkill at this exact moment because as far as I can see `de` is a fairly unique case until we add more payment processing options.

I'm making one more pass at this today as I've wrapped my head around Redux (mostly) and I think I've got an idea that will work for now and make expansion to an API endpoint at some point in the future entirely trivial.